schemes/kanagawa/kanagawa.nix

```
@@ -0,0 +1,24 @@
+{
+  slug = "Kanagawa";
+  name = "Kanagawa";
+  author = "rebelot";
```

```suggestion
  author = "Tommaso Laurenzi (https://github.com/rebelot)";
```
It's better if this is more detailed, like this